.travis.yml

```diff
@@ -10,6 +12,42 @@ stages:
 
 jobs:
   include:
+    - stage: push-build
+      language: node_js
+      node_js:
+        - stable
+      name: "Frontend"
+      before_script:
+        - cd src/frontend
+        - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.13.0
+        - export PATH="$HOME/.yarn/bin:$PATH"
+      cache:
+        yarn: true
+        directories:
+          - src/frontend/node_modules
+      script:
+        - yarn install --frozen-lockfile --check-files
+        - yarn run build
+
+    - name: "Backend"
+      language: java
+      jdk: openjdk10
+      before_cache:
+        - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+        - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+      cache:
+        directories:
+          - $HOME/.gradle/caches/
+          - $HOME/.gradle/wrapper/
+      services:
+        - postgresql
+      before_script:
+        - cd src/backend
+        - psql -c "CREATE DATABASE lpa;" -U postgres
+        - psql -c "CREATE USER lpa WITH PASSWORD 'example';" -U postgres
```

Comment:
I guess it's not needed here. However, if it is, then also add `- ./gradlew flywayClean flywayMigrate`